clang/include/clang/Basic/AttrDocs.td

```diff
@@ -480,6 +480,7 @@ following requirements.
 * Is not a C variadic function.
 * Is not a coroutine.
 * Is not defined as deleted or as defaulted.
+* Is not defined with a function try block.
```

Comment:
Did we submit an issue against the SYCL spec here?  Seems sensible to do (and could we do something like:)
```suggestion
* Is not defined with a function try block. See SYCL-CWG1234
```